core/modules/l2_forward.cc

```diff
@@ -562,7 +563,9 @@ void L2Forward::ProcessBatch(bess::PacketBatch *batch) {
 
     out_gates[i] = default_gate;
 
-    l2_find(&l2_table_, l2_addr_to_u64(snb->head_data<char *>()),
+    // read destination MAC address (first 6 bytes)
+    // NOTE: assumes little endian
```

Comment:
It's in `l2_forwad.h`

The little endian assumption is everywhere in the module, making it nontrivial to fix...